### PR TITLE
EJ2-69063 : How to change the default width and height of the PDFViewer. 

### PR DESCRIPTION
**Pull request by SaravanaPriya31**

### CODE REVIEW

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:2]: Layout: Post

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: How to change default size in EJ2 ASP.NET MVC PDF Viewer | Syncfusion
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:3]: Title: How to change the default size in EJ2 ASP.NET MVC PDF Viewer | Syncfusion

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: How to change default size in EJ2 ASP.NET MVC PDF Viewer | Syncfusion
+description: Learn here all about how to change the default width and height in ASP.NET MVC PDF Viewer component of Syncfusion Essential JS 2 and more.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:4]: Description: Learn how to change the default width and height in the ASP.NET MVC PDF Viewer component of Syncfusion Essential JS 2 and more.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: How to change default size in EJ2 ASP.NET MVC PDF Viewer | Syncfusion
+description: Learn here all about how to change the default width and height in ASP.NET MVC PDF Viewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:5]: Platform: ej2-asp-core-mvc

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: How to change default size in EJ2 ASP.NET MVC PDF Viewer | Syncfusion
+description: Learn here all about how to change the default width and height in ASP.NET MVC PDF Viewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to change the default width and height
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:6]: Control: How to change the default width and height

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: How to change default size in EJ2 ASP.NET MVC PDF Viewer | Syncfusion
+description: Learn here all about how to change the default width and height in ASP.NET MVC PDF Viewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to change the default width and height
+publishingplatform: ej2-asp-core-mvc
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:7]: Publishingplatform: ej2-asp-core-mvc

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: How to change default size in EJ2 ASP.NET MVC PDF Viewer | Syncfusion
+description: Learn here all about how to change the default width and height in ASP.NET MVC PDF Viewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to change the default width and height
+publishingplatform: ej2-asp-core-mvc
+documentation: UG
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:8]: Documentation: UG

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+description: Learn here all about how to change the default width and height in ASP.NET MVC PDF Viewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to change the default width and height
+publishingplatform: ej2-asp-core-mvc
+documentation: UG
+---
+
+# How to change the default width and height
+
+To change the default width and height of the Syncfusion PDF Viewer, you need to modify the `width` and `height` properties of the PDFViewer control.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:13]: To change the default width and height of the Syncfusion PDF Viewer, modify the `width` and `height` properties of the PDFViewer control.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+```html
+<div>
+    <div style="width:100%;height:600px">
+        @Html.EJS().PdfViewer("pdfviewer").ServiceUrl(VirtualPathUtility.ToAbsolute("~/Home/")).DocumentPath("PDF_Succinctly.pdf").Height("1000px").Width("800%").Render()
+    </div>
+</div>
+```
+
+In this example, the Width property is set to `800%` and the Height property is set to `1000px`. You can change these values to any desired size in pixels or percentage.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:23]: In this example, the Width property is set to `800%,` and the Height property is set to `1000px.` Change these values to any desired size in pixels or percentages.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+N> That the Syncfusion PDFViewer control requires a service URL that points to an endpoint that returns the PDF document to be displayed. In the example above, the ServiceUrl property is set to "~/Home/". You will need to modify this URL to match your own service endpoint that returns the PDF document.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/how-to/default-width-height.md:25]: N> The Syncfusion PDFViewer control requires a service URL that points to an endpoint that returns a PDF document to be displayed. In the example above, the ServiceUrl property is set to "~/Home/." Modify this URL to match your service endpoint that returns a PDF document.

**Review by KameshRajendran — APPROVED** [review]

**Pull request merged**